data/remnant/remnant 2 cognizance.txt

```diff
@@ -1200,7 +1200,7 @@ mission "Remnant: Cognizance 25"
 				`	"No."`
 					decline
 	npc kill
```

Comment:
If they aren't fanatics, perhaps it is enough just to stop these poor stranded fighters from snooping about, @Zitchas ?

```suggestion
	npc disable
```